.clang-format-ignore

```diff
@@ -6,3 +6,5 @@ src/Common/stb_image_write.h
 test/mobile_wall_hug/json.hpp
 externals/imgui_user/imgui_impl_sdl.h
 externals/imgui_user/imgui_impl_sdl.cpp
+externals/dr_libs/dr_mp3.h
```

Comment:
Why not use vcpkg?

Reply:
Because the package is not common on linux platforms and ti would break the system-deps